src/directives/layertree.js

```diff
@@ -170,20 +157,41 @@ ngeo.LayertreeController = function($scope, $element, $attrs) {
    * @type {number}
    * @export
    */
-  this.uid = goog.getUid(this);
+  this.depth = isRoot ? 0 : this.parent['depth'] + 1;
 
   /**
    * @type {number}
    * @export
    */
-  this.depth = isRoot ? 0 : this.parent['depth'] + 1;
+  this.uid = goog.getUid(this);
+
+  /**
+   * @type {Object|undefined} object containing available actions for the node
+   * @export
+   */
+  this.nodeActions = {
```

Comment:
same not sure here if it is usefull.
you can have `ng-if="::gmfLayertreeCtrl.depth == 1"`